packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template

```diff
@@ -83,7 +83,7 @@
     "@openapitools/openapi-generator-cli": "<%= versions['@openapitools/openapi-generator-cli'] %>",
     "@stylistic/eslint-plugin-ts": "<%= versions['@stylistic/eslint-plugin-ts'] %>",
     "@swc/cli": "^0.1.57",
-    "@swc/core": "^1.3.85",
+    "@swc/core": "1.5.7",
```

Comment:
should be retrieve from package.json like the other dependencies

Reply:
It should be, if `packages/@ama-sdk/schematics/package.json` had this dep :)

Reply:
https://github.com/AmadeusITGroup/otter/issues/1835